src/js/pages/AnswerQuestionsForm.jsx

```diff
@@ -95,18 +127,14 @@ const AnswerQuestionsForm = ({ classes, match }) => {
         <meta name="robots" content="noindex" data-react-helmet="true" />
       </Helmet>
       <PageContentContainer>
-        {questionnaire.questionnaireTitle && (
-          <TitleWrapper>
-            {questionnaire.questionnaireTitle}
-          </TitleWrapper>
-        )}
-        {questionnaire.questionnaireInstructions && (
-          <InstructionsWrapper>
-            {questionnaire.questionnaireInstructions}
-          </InstructionsWrapper>
-        )}
+        <TitleWrapper>
+          {questionnaire && questionnaire.questionnaireTitle}
```

Comment:
The reason we put the `{questionnaire.questionnaireTitle && (` outside the `<TitleWrapper>` styled div is because it has a margin-bottom that pushes down the page. If there is no  questionnaireTitle for example, we don't want that margin-bottom affecting the page.